index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>portfolio</title>
+</head>
+<body>
+  <header class="navigation">
+    <li class="logo">My Logo</li>
+    <li class="nav">
+      <img src="./image/Vector.svg" alt="">
+      <img src="./image/Vector.svg" alt="">
+      <img src="./image/Vector.svg" alt="">
+    </li>
```

Comment:
- [ ] Just like the other reviewer suggested, kindly wrap these in a `nav` tag. It is good to use the `nav` tag for semantic reasons and good user experiences. :+1:  The structure of this should be like this. Kindly implement it as the suggested way. :computer: 

```suggestion
     <nav>
         <ul class="nav">
            <li><img src="./image/Vector.svg" alt=""></li>
            <li><img src="./image/Vector.svg" alt=""></li>
            <li><img src="./image/Vector.svg" alt=""></li>
         </ul>
     </nav>
```